src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -114,17 +115,17 @@ We'll use the `<script>` tag to link to the Infusion library:
 <script type="text/javascript" src="lib/infusion/infusion-custom.js"></script>
 ```
 
-<div class="infusion-docs-note"><strong>Note:</strong> that the <code>infusion-custom.js</code> file is a concatenation of all of the required JavaScript files and will be minified (i.e. all of the whitespace removed) if you've built the minified version. If so, it might be difficult to debug with. If you want to be able to debug the code, you might want to choose the "source" version when you create your Infusion bundle.</div>
+<div class="infusion-docs-note"><strong>Note:</strong> The <code>infusion-custom.js</code> file is a concatenation of all of the required JavaScript files and will be minified (i.e. all of the whitespace removed) if you've built the minified version. If so, it might be difficult to debug with. If you want to be able to debug the code, you might want to choose the "source" version when you create your Infusion bundle.</div>
```

Comment:
This should reference both the Infusion-custom.js and the infusion-all.js files. In regards to debugging, another option would be to make use of the [source maps](https://github.com/fluid-project/infusion/blob/master/README.md#source-maps) generated with the build.